components/runtimes/charliecloud/SPECS/charliecloud.spec

```diff
@@ -99,8 +104,45 @@ EOF
 set     ModulesVersion      "%{version}"
 EOF
 
+cat > README.EL7 <<EOF
+For RHEL7 you must increase the number of available user namespaces to a non-
+zero number (note the number below is taken from the default for RHEL8):
+
+  echo user.max_user_namespaces=3171 >/etc/sysctl.d/51-userns.conf
+  reboot
+
+Note for versions below RHEL7.6, you will also need to enable user namespaces:
+
+  grubby --args=namespace.unpriv_enable=1 --update-kernel=ALL
+  reboot
+EOF
+
```

Comment:
Are user namespaces working on SLES 12 out of the box?

Reply:
I believe they do, however, let me check with a member of the cray team here to be certain.